termwidth.go

```diff
@@ -0,0 +1,7 @@
+// +build !linux
+
+package main
+
+func TermWidth() (int, bool) {
```

Comment:
nit: Benamse die return-parameter. Aus der Signatur+godoc-comment sollte klar sein, was die Funktion zu benutzen ist - in diesem Fall wäre z.B. "(width int, ok bool)" idiomatisch.